docker/puppetdb/docker-entrypoint.d/30-consul.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ "$CONSUL_ENABLED" = "true" ]; then
+  ipaddress="$(ifconfig eth0 | grep 'inet addr' | cut -d ':' -f 2 | cut -d ' ' -f 1)"
+  consul_hostname="${CONSUL_HOSTNAME:-consul}"
```

Comment:
In the puppetserver PR, we made these values `CONSULT_HOSTNAME` and `CONSUL_PORT` for consistency.

Reply:
@Iristyle updated